ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -555,3 +556,168 @@ def get_all_storageclass():
         )
     ]
     return storageclass
+
+
+def change_noobaa_endpoints_count(min_nb_eps=None, max_nb_eps=None):
```

Comment:
this code change doesn't belong here. Looks like a rebase thing

Reply:
So just delete it? I searched for this code in master and didn't find where is the source.